### PR TITLE
chore: Adds gitignore file for node project

### PR DESCRIPTION
**Pull request by AndrejsAbrickis**

### CODE REVIEW

File: .gitignore
Comment (abbasnake) [inline .gitignore:1]: Thinking weather to be a purist and request these comments to be deleted as well.

**Review by abbasnake — CHANGES_REQUESTED** [review]
**> chore: Adds gitignore file for node project**
Too much info, violation of rule 5, though it's not stated specifically. Issue - https://github.com/abbasnake/radical_divisions/issues/2

File: .gitignore
Comment (AndrejsAbrickis) [reply .gitignore:1]: agree

**Comment by AndrejsAbrickis** [comment]
Closing PR, as this repo, should NOT contain anything, except the README.md which also starts to feel like violating rule No.5
> 5. Absence of Documentation and Communication:
> Documentation, commit messages, and comments are disallowed.

**Pull request closed without merging**